mkosi

```diff
@@ -107,6 +107,7 @@ def create_image(args, workspace):
     print_step("Creating partition table...")
 
     f = tempfile.NamedTemporaryFile(dir = os.path.dirname(args.output), prefix='.mkosi-')
+    subprocess.run(["chattr", "+C", f.name], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, check=True)
     f.truncate(image_size(args))
```

Comment:
makes sense to add. And the patch is fine I figure, though I personally would always have tried to do the trivial ioctl this issues directly from python instead of shelling out to chattr...